src/source-napatech.c

```diff
@@ -115,6 +118,12 @@ SC_ATOMIC_DECLARE(uint16_t, numa1_count);
 SC_ATOMIC_DECLARE(uint16_t, numa2_count);
 SC_ATOMIC_DECLARE(uint16_t, numa3_count);
```

Comment:
I know this is preexisting ... Should stat support for platforms with numa nodes > 4 be considered/supported?